docs/source/documentation/team/wow-2.html.md.erb

```diff
@@ -0,0 +1,220 @@
+- **Product Manager (PM):** Oversees the product backlog, ensuring it aligns with stakeholder needs and business objectives
+- **Delivery Manager (DM):** Facilitates the Agile process, removing blockers and ensuring timelines are met
+
+## Communication and Delivery
+- **Daily Stand-ups:** A quick sync to share updates, plans, and blockers
+- **Ceremonies:** How we make sure our team stays aligned as we work to deliver during Agile Sprints.
+
+## Ceremonies
+
+### Schedule
+
+This team uses **Bi-weekly Sprints:** This is our timeframe for planning, executing, and reviewing work.
+
+The team has the following ceremonies:
+
+| Ceremony                                      | Occurence | Day             | Time          |
+| --------------------------------------------- | --------- | --------------- | ------------- |
+| [Stand Up](#stand-up)                         | Daily     | Monday - Friday | 10:00 - 10:15 |
+| [Backlog Management](#backlog-management)     | Weekly    | Thursday        | 14:00 - 15:00 |
+| [Refinement](#refinement)                     | Weekly    | Tuesday         | 11:00 - 12:00 |
+| [Sprint Planning](#sprint-planning)           | Bi-Weekly | Thursday        | 10:00 - 11:00 |
+| [Sprint Review](#sprint-review)               | Bi-Weekly | Thursday        | 10:00 - 11:00 |
+| [Sprint Retrospective](#sprint-retrospective) | Bi-Weekly | Tuesday         | 14:00 - 15:00 |
+| [Show & Tell](#show-and-tell)                 | Bi-Weekly | Thursday        | 15:00 - 15:30 |
+
+### Stand Up
+
+Our stand-ups run daily at 10:00 and last for fifteen minutes. We use [Slack Huddles](https://app.slack.com/huddle/T02DYEB3A/C04M8224WCV) and follow one of two formats:
+
+- **Walk The Board**
+
+Starting with the “Done” column we discuss any active issues on our [GitHub Projects Board](https://github.com/orgs/ministryofjustice/projects/27/views/18).
+
+- **Yesterday, Today, Blocked**
+
+A round robin of each team member discussing their issues and any blockers.
+
+### Backlog Management
+
+**Purpose:**
+Backlog Management is where we ensure our backlog is up to date and ready for refinement. It's where we:
+
+- Prioritise stories - Give a clear picture of what the team is working on, and will be working on next.
+- Remove stale stories - Keep the team focused on only relevant work.
+- Remove irrelevant stories - Prevent the team's focus from being split with work that's not required.
+
+**Process:**
+
+- TBD in consultation with PM/DM.
+
+**Output:**
+
+- A well prioritised backlog
+- A backlog that is ready for refinement and free of stale/irrelevant stories
+
+#### Audience
+
+- Product Manager
+- Delivery Manager
+- Leadership Team
+
+### Refinement
+
+**Purpose:**
+Refinement's our time to get on the same page about what we're tackling next. It's where we ensure everyone has a clear picture of the backlog items before they hit a sprint.
+
```

Comment:
How about "Refinement's is for the team to have an understanding and agreement of what we are working on in the next sprint. It's where we ensure everyone has a understanding of the backlog items before they are added to the sprint"